src/commonTest/kotlin/it/krzeminski/snakeyaml/engine/kmp/internal/TargetUtils.kt

```diff
@@ -0,0 +1,3 @@
+package it.krzeminski.snakeyaml.engine.kmp.internal
+
+expect fun areEnvVarsSupported(): Boolean
```

Comment:
suggestion: the name of the method says pretty much all but I think it would be good to add a comment why we need it (because env variables are not available in the browser)

Reply:
This is the `expect` declaration, and as such I'd refrain from leaking any impl details of concrete targets. E.g. with mentioning the browser we'd leak details about JS. It's true that we created this function solely to address no env vars in the browser, but in general we should keep things abstract. I can add some high-level comment here.